Added content Designing Bitcoin Products > Open Design (Issue #484) (#588)
BitcoinDesign/Guide: guide/designing-products/open-design.md

# Open design

**Open design welcomes the free exchange of ideas, knowledge, processes, and tools. It empowers by lifting barriers to innovation. It opens the door to scaling design projects by distributing adaptable resources.**

## A balance of independence and collaboration

> 

Reviewer: ```suggestion
```

> 

Reviewer: Fixed in https://github.com/BitcoinDesign/Guide/commit/453525b8045f957d98b6af6558586f2b80b7e200

Open source projects are often driven by global communities. These hubs of learning and building allow you to carve an individual niche. Align yourself to projects based on your passions, talents, interests, and beliefs. Your work becomes a voluntary public contribution toward common goals. Some projects are also supported through grants and donations.

> Open design communities can have multidisciplinary teams with designers, developers, copywriters, translators, project managers.... These diverse groups can develop stronger solutions through varied points of view.

Reviewer 2: Do you think it's worth mentioning that sometimes people take on multiple roles? And that you don't need to be trained or have professional experience to help out?

Reviewer 3: ```suggestion
Open design communities can have multidisciplinary teams with designers, developers, copywriters, translators, project managers, etc. These diverse groups can develop stronger solutions through varied points of view.
```

## Helpful skills

- Ethics
- Personal responsibility

> - Independence & self-will

Reviewer 2: I believe "self-will" is when someone is stubborn (meaning they do what they want no matter what anyone says). Do you mean "self-motivation" in that someone is driven on their own?

Reviewer 3: Agreed. I'm actually not familiar with the term "self-will" but "self-motivation" sounds right.

Reviewer: Changed to self-motivation in https://github.com/BitcoinDesign/Guide/commit/453525b8045f957d98b6af6558586f2b80b7e200

- Communication
- Open-mindedness
- Respectful feedback
- Time management
- Teamwork
- Openness & transparency

## Bitcoin has a spirit of openness

> We could argue Bitcoin needs open design to stay true to its mission.

Reviewer 2: If we think of design as "how it works", then the (technical) design of Bitcoin is already rooted in openness. We just need to make sure the application layer, the layer that people actually interface with, also shares the same values.

Reviewer: @GBKS So what do you recommend, any specific suggestion?

* Bitcoin is public and open source

> * It has public & unrestricted network participation

Reviewer 3: ```suggestion
* It has public and unrestricted network participation
```

> * It allows permissionless transfer of value in the Bitcoin protocol

Reviewer: ```suggestion
* It allows permissionless transfer of value
```

Thousands of projects have borrowed ideas, techniques, and code from Bitcoin. Countless open-source experiments have arisen. To a designer, Bitcoin is a robust foundation for collaboration. A foundation that comes with a responsibility toward the benefit of society.

> This ethical responsibility encompasses application design. Most people interact with applications instead of the underlying Bitcoin technical protocol. If someone controls access to an application, it won't matter if the protocol is open. It is up to the global community to build principled Bitcoin applications.

Reviewer: ```suggestion
This ethical responsibility encompasses application design. Most people interact with applications instead of the underlying Bitcoin technical protocol. If someone controls access to an application, it won't matter if the protocol is open. It is up to the global community to build [principled Bitcoin applications](guide/designing-products/principles/)
```

> Bitcoin is a protocol created to give access to anyone. Access and accessibility go hand in hand. Designing Bitcoin products means working on removing barriers of entry. It means making decisions to create a unique balance of features for ease of use. You should design for a specific audience at the same time as giving the widest access.

Reviewer 2: > You should design for a specific audience at the same time as giving the widest access.

This is a little confusing.

Reviewer 3: Agreed. I was a little confused as well. @AlexaAker what are you hoping to convey here?

Reviewer: Removed the sentence in https://github.com/BitcoinDesign/Guide/commit/453525b8045f957d98b6af6558586f2b80b7e200

## Values

Open-source software development has existed for decades. Public review and debate have helped it create better products and services. Open design shares similar values, though it is not as known or practiced.

### Integrity

> - Make moral and responsible choices

Reviewer 3: These bullet points (complete sentences) have periods at the end. Other bullet points (also complete sentences) lack periods on the end. We should probably pick one and be consistent.

The page on the live site has this same inconsistency; we could also unify it in a separate PR.

Reviewer: @sbddesign  There are no periods at the end here? Are you sure you're referring to this paragraph/line?

- Build with a principled foundation

### Collaboration

- Collaborate without competing
- Approach problems with an open-mind

> - Share transparent work and open processes

Reviewer 2: ```suggestion
- Share your work and process in an open and transparent way
```
"transparent work" reads like the work is transparent. Did you mean to say the the sharing should be done in a transparent way?

- Make it easy for others to build on our work and ideas
- Be considerate while giving and receiving feedback

### Inclusion

- Aim for inclusiveness, usability, and accessibility

> - Design features for the broadest range of people, locations, backgrounds, technological knowledge, financial literacy...

Reviewer 3: ```suggestion
- Design features for the broadest range of people, locations, backgrounds, technological knowledge, financial literacy, etc.
```

- Embrace multidisciplinary feedback

### Real world solutions

- Help design products that solve real problems and address real needs

> - Rely on our peers for quality assurance and validation

Reviewer 2: ```suggestion
- Rely on your peers for quality assurance and validation
```

- Fulfill the open design group's goals through actions
- Create functional elements that have a positive influence on how people feel and think

## Discover the Bitcoin Design Community

**Our focus is on exploring best practices to improve Bitcoin product design.**

- Join the conversation on [Slack]({{site.slack_invite_url}}). Navigate toward the channels that interest you. Dive into topics and conversations that draw you in.

> - Identify your strengths and goals. Figure out how they may fit within the community's vision. A good place to start is listening to a [Community Call](https://github.com/BitcoinDesign/Meta/issues?q=is%3Aissue+is%3Aopen+Community+Call) or reading our [newsletter](https://bitcoindesign.substack.com/).  

Reviewer 2: How about following on Twitter, and also linking the YouTube channel? There's a big collection of videos that may give people a good idea of how the calls pan out. Even if they can't join due to time zones, work, family, etc, they can still get a sense from the videos.

Reviewer: Done in https://github.com/BitcoinDesign/Guide/pull/588#discussion_r766540497

- Feel free to ask questions or ask for help in solving bitcoin design issues.
- See some of the community [Projects](https://bitcoin.design/projects/).
- Visit our [Contribute](https://bitcoin.design/contribute/) page.

### Test, redesign, and engage in feedback

- Experiment with Bitcoin and Lightning. Download a wallet and test small amounts. You can also test network features without using real bitcoin on Testnet. It works with some wallets.

> - Redesign a bitcoin and/or lightning application or focus on a CX issue that interests you. Even as a brief exercise, a redesign is a great way to develop understanding of how something works.

Reviewer: ```suggestion
- Redesign a bitcoin and/or lightning application or focus on a UX issue that interests you. Even as a brief exercise, a redesign is a great way to develop understanding of how something works.
```

Reviewer 2: ```suggestion
- Redesign a bitcoin and/or lightning application or focus on a user experience issue that interests you. Even as a brief exercise, a redesign is a great way to develop understanding of how something works.
```
Abbreviations require too much brain power :)

- Join a [Design Review](https://github.com/BitcoinDesign/Meta/issues?q=is%3Aissue+is%3Aopen+Design+Review+Call+). A call focusing on giving feedback to improve a specific bitcoin product. Products vary by call.

> - Join the [Wallet Improvement Project](https://github.com/BitcoinDesign/Guide/issues/493). It tests a list of real world bitcoin projects and suggests redesigns to their makers.  

Reviewer: ```suggestion
- Join the [Wallet Improvement Project](https://github.com/BitcoinDesign/Guide/issues/493). It tests a list of real world Bitcoin projects and suggests redesigns to their makers.  
```

### Continuous learning

> Read, watch, listen, and test. You can discover great material about Bitcoin and related topics (like finance). You may not find as much material on the design side of bitcoin applications.

Reviewer 2: Would be cool if we had some starting points, like a YouTube playlist with good design-related videos, or a few podcast episodes to link to.

Reviewer: Do we really have such resources could you provide a specific suggestion with the links, as I'm not really 100% sure what to add here.

### Open up resources

- Try new ways of making your knowledge, tools, and workflows public and accessible to others.

> - Create free and open resources like an [icon set](https://bitcoinicons.com/), a [UI kit](https://www.bitcoinuikit.com/), a workshop....

Reviewer 3: ```suggestion
- Create free and open resources like an [icon set](https://bitcoinicons.com/), a [UI kit](https://www.bitcoinuikit.com/), or a workshop.
```
